### DISCUSSION TITLE
中央通識修課經驗-資訊與網路素養

### DISCUSSION THREAD

**Discussion by Cerosop** [1101-GS4719-C/Python-Community-GS4719-C#1780] [General]
資訊與網路素養-連文雄 

這堂課雖然需要點名但是也是很輕鬆，上課會教一些關於資料以及excel等等的操作，每周結束都會有簡單的作業要做，大概花20分鐘就可以寫完了，也有ppt可以讓你考前好好複習，ppt內容也沒到太多，負擔很輕，這堂課有期中跟期末考，各10題問答題，但問的內容通通都在ppt裡，我認為不會很難，最終成績95，成績很甜，也能學到實用的知識

**Comment by davidteng00000** [comment]
我有聽我去修這堂課的同學說他都沒去然後期中考完全沒讀也拿高分，很推這堂課
